config/credentials.yml.enc

```diff
@@ -1 +1 @@
-ZK0DaRjyBykIvHBoUw3E/sXlkClJ8S5qalq6mQ2+A/2pN0xssPLvzfFXZWTzDzd7rMHvmgO5OXI1ZEwTUOB/evwrQmgLdyWMXg140jiOT1YnSThNcjo+I+YlMkJcMTV+F4atNMod2dj+omyE0K+l342WyoScFmwB9CwApXniomjiIwAMOR/QTKJOTfbQ1k2qfL27mTUb0n6LkkKpzk6wxQRtAen4vfjTSI9UQ1ollILw085Q52OS7IdtvxG+xd52PutEJ8vYAfy1rRWYhrFPBN3iNkPm9MxDoYl6mw53ee/kBjsml2z1MGTFy7cIAxpJotyOWnrFt5NkFXiKkqlesbE1qxWYIiDhqH+JNEbVILXVYch9YZSGzZGM9jO2gkawEPdP9vEyPMMdXaLET+qgWydqda/cE19tyFowpNQg+3+0HnyNnXIEwsBJRVrBdYAdvozYJMaQQO0EGshEF33s1iv2wdKaaJqkhshdnHjey2qKur7k/QyyOQ==--yTNEtmHYf8KC1Ue2--hZPvnbk58hQWnbmIbPixYA==
\ No newline at end of file
+JYMYdQJ1Ro67iip3/KCHCB4/8bpel1G2c/kBZNkIeKVCI0O3KXRElPaCuKdgMdp4fHCHiMK3ghXzda7ZIkhyEqYDHVHELIf1zVUBSL+PTM4z9bDF6d9PWK80wE1wh5Frn8ikLzuy05w6QBWhBmm4lW7IQpmdhRmUN78MyYSOxBZrk6ua0NrHaOV0RqiCvQuCkNsZHqN9GBrVrJDyn12azKiDokpB748HEqjaurBClsip/kkpMhMb/2x1AHky6WzDS3TLk5CLCoS7gMuDYaRSqHTmwu9ingUxnu5VWZkpt4egoA4XHmG9RnszLu5tuusKvWNG6A7gUG0C56lNWf/4IloIPadw0nC9fNaWIcO0MLGQIzzgXMEauWySMFriw8UD0JoGwXMDgDJA5EsA2znIE28m2sDUKbzd+bu/Om1sU9NVdG6PyN92rarq/EJ5JMHgYFOi9O7bFSvGgXfwwZ4ZdjkTlhxqBA3SdI1e4VAPeJqNzB3JUtijDg==--DBjlpfwVbo+l1CCn--d+MTjt832Ml+jQmejijLRQ==
```

Comment:
There were no pending changes to the credentials, so updating the api key looks fine :)